### PR TITLE
Split in 2

### PR DESCRIPTION
**Pull request by SimenB**
Missing README and stuff, other than that it should be GTG

/cc @sverosak @peternic @CMTegner

### CODE REVIEW

File: packages/eslint-config-finn/es-modules.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'use strict';
+module.exports = {
+    extends: [
+        'eslint-config-spt/env-es6-modules',
```
Comment (SimenB) [inline packages/eslint-config-finn/es-modules.js]: has to be full name for the resolution logic to work

**Comment by sveisvei** [comment]
Could not see any issues at a glance, wfm

**Comment by CMTegner** [comment]
What's the reason for this change?

**Comment by SimenB** [comment]
So that projects not using react don't have to include the react plugin. Matches schibsted/eslint-config-spt#12

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (SimenB) [inline .eslintignore]: needed because eslint only filters out `node_modules` in cwd, not nested ones

**Pull request merged**